setup.py

```diff
@@ -15,6 +15,6 @@
     packages=find_packages(),
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
-    install_requires=["Flask>=1.0.2", "user-agents>=2.0.0", "limits>=3.2.0"],
+    install_requires=["Flask>=1.0.2", "user-agents>=2.0.0", "Flask-Limiter>=1.4"],
```

Comment:
I think Flask-Limiter requires a higher version of Flask. So to keep this simple, can you update `flask==1.1.2`

Reply:
Wait what version is this? Last version of [Flask-limiter is 3.7](https://pypi.org/project/Flask-Limiter/)?

Reply:
This is because Flask-Limiter leverages 'ensure_sync' that wasn't introduced into Flask until 2.0